app/services/service_listeners/attachment_link_header_updater.rb

```diff
@@ -0,0 +1,43 @@
+module ServiceListeners
+  class AttachmentLinkHeaderUpdater
+    include Rails.application.routes.url_helpers
+    include PublicDocumentRoutesHelper
```

Comment:
Minor: I'm not sure either of these `include` statements are needed are they? I have a feeling they've been unnecessarily copied & pasted into a number of the updater classes.

Reply:
Good spot. I've removed them.